supplementary/devices/wt88047.rst

```diff
@@ -24,14 +24,12 @@ Distributions
      - What works
      - What doesn't work
    * - `WT88047 on LineageOS Wiki <https://wiki.lineageos.org/devices/wt88047>`_
-     - `LineageOS\android_device_wingtech_wt88047 <https://github.com/LineageOS/android_device_wingtech_wt88047>`_
-     - `LineageOS\android_device_wingtech_msm8916-common <https://github.com/LineageOS/android_device_wingtech_msm8916-common>`_
+     - `LineageOS\android_device_wingtech_wt88047 <https://github.com/LineageOS/android_device_wingtech_wt8804>`_
      - `LineageOS\android_kernel_wingtech_msm8916 <https://github.com/LineageOS/android_kernel_wingtech_msm8916>`_ based on v3.10.49
      - see device page
      - see device page
    * - `WT88047 on Halium projectmanagement <https://github.com/Halium/projectmanagement/issues/36>`_
```

Comment:
Please remove this: there should be only LineageOS and other ports, not halium itself.

Reply:
The problem is,  it's also wrong on the pme's page, not your fault.

Reply:
@LNJ2 noted, will fix that as soon as i can